### PR TITLE
Update TEAMS.md

### PR DESCRIPTION
**Pull request by jkutner**
Adding @joshwlewis as Distribution team contributor

### CODE REVIEW

File: TEAMS.md [hunk 1]
```diff
@@ -44,7 +44,7 @@
 
 ### Project Contributors
 
-[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@mritunjaysharma394][@mritunjaysharma394]
+[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@mritunjaysharma394][@mritunjaysharma394], [@joshwlewis][@joshwlewis]
```
Comment (jromero) [inline TEAMS.md:47]: nit: we should probably keep them alphabetical.

Comment (hone) [inline TEAMS.md:47]:
```suggestion
[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@joshwlewis][@joshwlewis], [@mritunjaysharma394][@mritunjaysharma394]
```

Comment (hone) [reply TEAMS.md:47]:
```suggestion
[@AswinTimalsina][@AswinTimalsina], [@Brahyt][@Brahyt], [@joshwlewis][@joshwlewis], [@mritunjaysharma394][@mritunjaysharma394]
```

**Review by hone — APPROVED** [review]

**Comment by joshwlewis** [comment]
Neat. Thanks! 💜

**Pull request merged**